packages/quick-edit/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@cloudflare/quick-edit",
+	"version": "0.0.0",
+	"private": true,
+	"description": "VSCode for Web hosted for use in Cloudflare's Quick Editor",
+	"homepage": "https://github.com/cloudflare/workers-sdk#readme",
+	"bugs": {
+		"url": "https://github.com/cloudflare/workers-sdk/issues"
+	},
+	"license": "BSD-3-Clause",
+	"author": "workers-devprod@cloudflare.com",
+	"scripts": {
+		"build": "./build.sh",
+		"dev": "concurrently './vscode/scripts/code-web.sh --extensionDevelopmentPath ../web/quick-edit-extension' 'npm --prefix web/quick-edit-extension run watch-web' 'yarn --cwd vscode watch' 'yarn --cwd vscode watch-web'",
```

Comment:
Checking the port 8080 (which it needs to use) is occupied and then clearing the port, could be useful.